pkgs/development/libraries/qt-6/modules/qtmultimedia.nix

```diff
@@ -40,6 +40,8 @@ qtModule {
   patches = lib.optionals stdenv.hostPlatform.isMinGW [
     ../patches/qtmultimedia-windows-no-uppercase-libs.patch
     ../patches/qtmultimedia-windows-resolve-function-name.patch
+  ] ++ [
```

Comment:
I am confused why this PR is about mingw but the patch is not under the condition.

Reply:
Looks like to me that this patch is required for mingw, but is a general fix, so it can be applied always. Doing the latter prevents bitrot.